Core/Src/main.c

```diff
@@ -3,21 +3,37 @@
 #include "pins.h"
 #include "main.h"
 #include <string.h>
+#include <stdio.h>
```

Comment:
It is preferable to add standard library includes at the beginning of the block of includes